tests/app/forms/field_handlers/test_address_handler.py

```diff
@@ -6,8 +6,11 @@
 from app.forms.field_handlers import AddressHandler
 
 
-def get_test_form_class(
-    answer_schema, value_source_resolver, rule_evaluator, messages=error_messages.copy()
+def get_test_form_class(  # pylint: disable=dangerous-default-value
+    answer_schema,
+    value_source_resolver,
+    rule_evaluator,
+    messages=error_messages.copy(),
 ):
```

Comment:
I would resolve this.

Should just be:

```py
def get_test_form_class(
    answer_schema,
    value_source_resolver,
    rule_evaluator,
    messages=None,
):

  if not messages:
    messages = error_messages.copy()
```

Reply:
Made the suggested changes 👍